sql/parser/eval.go

```diff
@@ -441,15 +589,27 @@ var binOps = map[binArgs]func(Datum, Datum) (Datum, error){
 
 	// TODO(pmattis): Overflow/underflow checks?
 
-	// TODO(pmattis): Should we allow the implicit conversion from int to float
```

Comment:
Perhap it needs to be reworded. The implicit casts were removed in

https://github.com/cockroachdb/cockroach/pull/1793

Did you have anything specific in mind?